baybe/targets/numerical.py

```diff
@@ -120,11 +120,22 @@ def _validate_transformation(  # noqa: DOC101, DOC103
                 f"of {_VALID_TRANSFORMATIONS[self.mode]}."
             )
 
+    @property
+    def is_normalized(self) -> bool:
+        """Indicate if the computational representation lies in the unit interval."""
+        return self.bounds.is_bounded and self.transformation is not None
```

Comment:
Although not strictly necessary, I would prefer parantheses here for better readability.

Reply:
Your wish is my command =)